examples/new_design_examples/new_design_msv.m

```diff
@@ -0,0 +1,98 @@
+clear all
+close all
+import casadi.*
+import vdx.*
+
+R = 3.5;
+%% Define projected system
+problem_options = nosnoc.Options();
+solver_options = nosnoc.solver.Options();
+%problem_options.rk_scheme = RKSchemes.GAUSS_LEGENDRE;
```

Comment:
this option does not work properly. extremely bad convergence. I guess it is not implemented properly because of the rbp?
In case yes, throw a warning and put radau iia.

But in general it might not have be bad to have GL. The resulting "sliding modes" ODEs are not of very high index like in CLS problems, hence they have decent properties.

Reply:
:+1:  I am happy to tear out the current implementation as I agree it is not ready for `main.